app/views/spotlight/catalog/_openseadragon_default.html.erb

```diff
@@ -0,0 +1,57 @@
+<% image = document.to_openseadragon(blacklight_config.view_config(:show)) %>
+<%
+	image = []
+	@docs.each do |doc|
+		image << doc.to_openseadragon(blacklight_config.view_config(:show)).first.first.first
```

Comment:
If we changed `document.to_openseadragon` to implement this behavior, would that allow us to avoid overriding this template?